integration-tests/tests/src/hooks/import-app-before.ts

```diff
@@ -30,6 +32,14 @@ export function importAppBefore(
     } else {
       this.app = await importApp(name, replacements);
       Realm.App.Sync.setLogLevel(this.app, logLevel);
+      // Set a default logger as Android does not forward stdout
+      Realm.App.Sync.setLogger(this.app, (level, message) => {
+        const magentaDate = `\x1b[35m${new Date().toISOString().replace("T", " ").replace("Z", "")}`;
```

Comment:
The full date timestamps feel a bit excessive to me. Perhaps an HH:MM:SS.ms would be enough?

Nit: I'd personally split this in two lines, I don't like debugging expressions inside of template strings (same goes for the line below).

Reply:
Yea fair point maybe the data isn't that useful!